src/services/ErrorService.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import ErrorResponse from '../models/ErrorResponse'
+import _ from 'lodash'
+
+const develop = process.env.NODE_ENV === 'development'
+
+export default class ErrorService {
+  createInternalServerErrorResponse (error: Error): Error | ErrorResponse {
+    console.error(error)
+    if (develop) {
+      return error
+    } else {
+      return new ErrorResponse('server',
+        'Ein interner Serverfehler ist aufgetreten. Bitte kontaktieren Sie Ihren Administrator.')
+    }
+  }
+
+  createOfferNotFoundErrorResponse (token: string): ErrorResponse {
+    return new ErrorResponse('token', `Das Angebot mit Token '${token}' existiert nicht oder wurde gelöscht.`)
```

Comment:
Maybe we can create an object with all ErrorTypes, like:
```js
const errorTypes = {
  token: 'token',
  confirmation: 'confirmation',
  validation: 'validation'
}
```
to avoid missspellings